sites/programacion.tcc.com.uy/programacion.tcc.com.uy.config.js

```diff
@@ -0,0 +1,102 @@
+const axios = require('axios')
+const dayjs = require('dayjs')
+
+const API_ENDPOINT = 'https://www.tccvivo.com.uy/api/v1/navigation_filter/1575/filter/'
+
+module.exports = {
+  site: 'programacion.tcc.com.uy',
+  days: 2,
+  request: {
+    cache: {
+      ttl: 60 * 60 * 1000 // 1 hour
+    },
+    maxContentLength: 10 * 1024 * 1024 // 30Mb
+  },
+  url: function ({ date }) {
+    return `${API_ENDPOINT}?cable_operator=1&emission_start=${date.format(
+      'YYYY-MM-DDTHH:mm:ss[Z]'
+    )}&emission_end=${date.add(1, 'd').format('YYYY-MM-DDTHH:mm:ss[Z]')}&format=json`
+  },
+  parser({ content, channel }) {
+    let programs = []
+    let items = parseItems(content, channel)
+    items.forEach(item => {
+      programs.push({
+        title: parseTitle(item),
+        description: parseDescription(item),
+        categories: parseCategories(item),
+        date: item.year,
+        season: item.season_number,
+        episode: item.episode_number,
+        icon: parseIcon(item),
+        start: parseStart(item),
+        stop: parseStop(item)
+      })
+    })
+
+    return programs
+  },
+  async channels() {
+    const data = await axios
+      .get(
+        `${API_ENDPOINT}?cable_operator=1&emission_start=${dayjs().format(
+          'YYYY-MM-DDTHH:mm:ss[Z]'
+        )}&emission_end=${dayjs().format('YYYY-MM-DDTHH:mm:ss[Z]')}&format=json`
+      )
+      .then(r => r.data)
+      .catch(console.error)
+
+    return data.results.map(c => {
+      return {
+        lang: 'es',
+        site_id: c.id,
+        name: c.name.replace(/^\[.*\]\s/, '')
+      }
+    })
+  }
+}
+
+function parseTitle(item) {
+  const localized = item.localized.find(i => i.language === 'es')
+
+  return localized ? localized.title : null
```

Comment:
Not sure it makes a huge difference but could use original_title as a fallback if no localised title
```suggestion
  return localized ? localized.title : item.original_title
```